docs/kubernetes/README.md

```diff
@@ -123,6 +123,21 @@ parameters:
 
 Your nodes might need to have `cryptsetup` installed to mount the volumes with LUKS.
 
+### Formatting Options
+
+You can specify extra formatting options which are passed directly to `mkfs.FSTYPE` via the `formatOptions` parameter in the storage class.
+
+#### Example
+```yaml
+parameters:
+  csi.storage.k8s.io/fstype: xfs
+  formatOptions: "-i nrext64=1"
```

Comment:
Should this parameter be prefixed with `fs` ? `fsFormatOptions` or even `fsFormatArgs` ?

Reply:
How about `mkfsFormatOptions`? Then we are very verbose about whats actually happening under the hood. Might also easier for an user to identify what options are supported (going directly to the `mkfs` docs).